tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -613,6 +613,10 @@ decap/test_decap.py:
     reason: 'Skip on t1-isolated-d32/128 topos'
     conditions:
       - "topo_name in ['t1-isolated-d128', 't1-isolated-d32']"
+  xfail:
```

Comment:
Hi Anton, this change might not be needed, because decap test module is not in the included list.

Reply:
the qual doc has a include list (not a exclude list), which does not has decap module it.